src/app/tests/suites/certification/Test_TC_DA_1_9.yaml

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) 2025 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: Add the test cases for indirect CRLs
+
+name: 29.1.9. [TC-DA-1.9] Device Attestation Revocation [DUT-Commissioner]
+
+PICS:
+    - MCORE.ROLE.COMMISSIONER
+
+config:
+    nodeId: 0x12344321
+    cluster: "Basic Information"
+    endpoint: 0
+
+tests:
+    - label: "Precondition/Test case description"
+      verification: |
+          (for all steps, PIs use equivalent command on their respective platform)
+          Validate that the DUT properly handles revoked device attestation certificates during commissioning, including certificates revoked through indirect CRLs.
+          set the $CHIP_ROOT environment variable to the location of the connectedhomeip directory (cd to the connectedhomeip directory, use command `export CHIP_ROOT=$(pwd -P)`
```

Comment:
This stuff is too platform specific. Why is it needed

Reply:
inherited from TC_DA_1_8, added the python test for TC_DA_1_9 based on #28762. Please check that.